tests/test_utils.py

```diff
@@ -141,8 +142,14 @@ def test_bcpandas_creates_command_with_encrypt_yes(run_cmd):
             "me",
             "-P",
             "secret",
-            "-Ym",
-        ],
+        ]
+        + (
+            [
+                "-Ym",
```

Comment:
Black will format much nicer without the comma 😄 
```suggestion
                "-Ym"
```